iree/modules/check/native_module.cc

```diff
@@ -83,6 +84,20 @@ bool AlmostEqByteSpan(iree_byte_span_t lhs_bytes, iree_byte_span_t rhs_bytes) {
   return true;
 }
 
+bool AlmostEqByteSpanF16(iree_byte_span_t lhs_bytes,
+                         iree_byte_span_t rhs_bytes) {
+  auto lhs_span = AbslSpan<uint16_t>(lhs_bytes);
+  auto rhs_span = AbslSpan<uint16_t>(rhs_bytes);
+  assert(lhs_span.size() == rhs_span.size());
+  for (int i = 0; i < lhs_span.size(); ++i) {
+    if (fabs(half_float::detail::half2float<float>(lhs_span[i]) -
+             half_float::detail::half2float<float>(rhs_span[i])) > 0.0001) {
```

Comment:
Let's extract this into a constant, since now we're using it in multiple places

Reply:
Sure changed it, however I'm suspecting we might want a different value for f16 at some point as we may have higher precision difference.